native/Cargo.toml

```diff
@@ -39,15 +39,15 @@ arrow-buffer = { version = "54.0.0" }
 arrow-data = { version = "54.0.0" }
 arrow-schema = { version = "54.0.0" }
 parquet = { version = "54.0.0", default-features = false, features = ["experimental"] }
-datafusion = { git = "https://github.com/apache/datafusion", rev = "5592834", default-features = false, features = ["unicode_expressions", "crypto_expressions"] }
-datafusion-common = { git = "https://github.com/apache/datafusion", rev = "5592834", default-features = false }
-datafusion-functions = { git = "https://github.com/apache/datafusion", rev = "5592834", default-features = false, features = ["crypto_expressions"] }
-datafusion-functions-nested = { git = "https://github.com/apache/datafusion", rev = "5592834", default-features = false }
-datafusion-expr = { git = "https://github.com/apache/datafusion", rev = "5592834", default-features = false }
-datafusion-expr-common = { git = "https://github.com/apache/datafusion", rev = "5592834", default-features = false }
-datafusion-execution = { git = "https://github.com/apache/datafusion", rev = "5592834", default-features = false }
-datafusion-physical-plan = { git = "https://github.com/apache/datafusion", rev = "5592834", default-features = false }
-datafusion-physical-expr = { git = "https://github.com/apache/datafusion", rev = "5592834", default-features = false }
+datafusion = { git = "https://github.com/apache/datafusion", rev = "8d006a2", default-features = false, features = ["unicode_expressions", "crypto_expressions"] }
```

Comment:
For my edification, is there a reason why we have to use a git revision. Is this not published to a public repository?

Reply:
DF 45 is not released yet, but will be in the next week or so. The goal of this and the previous PR is to incrementally update so that we are ready once the release is published.

Reply:
@parthchandra the PR is now updated to use the official release